data/enemy_packs.py

```diff
@@ -37,6 +42,10 @@ def __init__(self, rom, args, formations):
 
     def _replaceable_bosses(self):
         replaceable = list(bosses.normal_pack_name)
+        if not self.args.mix_bosses_statues:
+            for boss in [self.DOOM, self.GODDESS, self.POLTERGEIST]:
+                replaceable.remove(boss)
+                self.event_boss_replacements[boss] = boss
```

Comment:
Should this flag also be applied in randomize_packs? That is, if this flag is disabled should the statues also be prevented from appearing in fixed/random encounters?

Reply:
Just now getting back to this!

I personally feel as long as the statues are being scaled with the other bosses them showing up in the encounter is fine.

Having said that, it doesn't look like the dragons can be chosen when they are shuffled with bosses - Probably explicitly checking the flag in randomized_packs makes the sense

I'll go ahead and make the update